submissions/kosh3n/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link type="image/png" sizes="16x16" rel="icon" href="https://img.icons8.com/fluency/48/000000/circled-menu.png" />
+</head>
+
+<body>
+    <header>
+        <div class="logo-conatiner">
+            <a href="#home"><img class="homo-img" src="https://img.icons8.com/parakeet/96/000000/experimental-home-parakeet.png" alt="home"></a>
+        </div>
+        <div class="welcome">
+            <p>Welcome to our site!</p>
+        </div>
+
+            <nav class="navigation">
+                <!-- icon">  -->
+                <label for="hamburger">
+                    <span tabindex="0"> <img src="dots-menu.png" alt="dots-menu.png"></span></label>
+                <input type="checkbox" id="hamburger">
+                <ul class="nav-menu" tabindex="0">
+                    <!-- this section is shown -->
+                    <li><a href="#mail">
+                            <figure>
+                                <img src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
+                                <figcaption>Mail</figcaption>
+                            </figure>
+                        </a></li>
+                    <li><a href="#">
```

Comment:
```suggestion
                    <li>
                        <a href="#mail">
                            <figure>
                                <img src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
                                <figcaption>Mail</figcaption>
                            </figure>
                        </a>
                    </li>
                    <li>
                        <a href="#">
```
Please, format your file to make it more readable